### ISSUE TITLE
Add Gaming section in the search filter menu

### ISSUE THREAD

**Issue by stealthd3v** [TeamNewPipe/NewPipe#6698]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
It would be great if we can add a **Youtube Gaming** section similar to **Youtube Music** in the search filter menu.

#### Is your feature request related to a problem? Please describe it
My feature request is not related to any problem,just an additional enhancement to the category section in the search feature menu. We can reduce the long list by adding the sections as collapse section lists.

#### Additional context
<img src="https://i.imgur.com/Q4k03OE.png" height="600px" width="350px">

#### How will you/everyone benefit from this feature?
This will help the gamers to search filtered gaming content. Can also be further added with another feature that can include Sports,Fashion,Learning etc section in the which can be further added with collapse section list. This will help the other people if further added with other sections.

**Comment by mhmdanas** [comment]
Personally, I think this dialog is not fitting for this feature. YouTube Music is there on the dialog because it is quasi-separate from YouTube (e.g. has separate official interface), while gaming videos are just one kind of video. I believe your request would be also fulfilled with https://github.com/TeamNewPipe/NewPipe/issues/1805. What do you think?

**Comment by stealthd3v** [comment]
> Personally, I think this dialog is not fitting for this feature. YouTube Music is there on the dialog because it is quasi-separate from YouTube (e.g. has separate official interface), while gaming videos are just one kind of video. I believe your request would be also fulfilled with #1805. What do you think?

Yeah,I think this feature would be a fit, similar to the the issue #1805. But I was actually thinking that this will be more for search filter for different categories and the issue #1805 will be for the trending page categories.

**Comment by mhmdanas** [comment]
Good point, but I still think that this overflow menu is not fitting for such a feature. Perhaps if searching by tags was added, one could search for #gaming (although at this point I'm not sure if this would be better than just searching for gaming directly).

**Comment by stealthd3v** [comment]
Yeah,right. Agreed on this one. It can be later,searching by tags is also a good one to think about.
